test/sql/aggregate/aggregates/test_median.test

```diff
@@ -23,62 +23,57 @@ statement ok
 create table quantile as select range r, random() from range(10000) union all values (NULL, 0.1), (NULL, 0.5), (NULL, 0.9) order by 2;
 
 query I
-SELECT median(r) FROM quantile
-----
-4999.5
-
-query I
-SELECT median(r) FROM quantile
+SELECT median(r)::VARCHAR FROM quantile
```

Comment:
Can we remove these cases/merge them into the `foreach type <integral>` cases you have below? They seem redundant to each other.

Reply:
This test also checks that NULLs are ignored (notice that there are 3 of them so the count becomes even.

Reply:
Ah, yes the NULLs are good to test for. Just to be clear, the total count is 10,003 (odd), but the count of non-nulls is even.

This actually makes me realize we never test for the odd-count case. Should I submit a followup PR that adds tests for that? Also to make sure that the result type is floating, eg`MEDIAN([1,2,3])::VARCHAR` is "2.0". This isn't super obvious, if you haven't thought about the even-count case you might expect the type of the median to be the same type as the input.

Regardless, this PR seems good to go without this change.